### PR TITLE
Add SPDX header - batch 16

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Resolves: last remaining SPDX header for non-dist-folder items

## Summary

* files with copyright information: 10957 / 11065
* files with license information: 10947 / 11065

## TODO

- [x] green CI
- [x] review and merge

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/tests/UserMigration/assets/account.png
Comment (susnux) [inline apps/settings/tests/UserMigration/assets/account.png:1]: :heart_eyes: :beach_umbrella:

File: apps/webhook_listeners/img/app-dark.svg
Comment (susnux) [inline apps/webhook_listeners/img/app-dark.svg:1]: Much better to not confuse with workflow

**Review by susnux — APPROVED** [review]

File: apps/settings/tests/UserMigration/assets/account.png
Comment (AndyScherzinger) [reply apps/settings/tests/UserMigration/assets/account.png:1]: needed a picture that I know the actual license

**Review by artonge — APPROVED** [review]

**Comment by susnux** [comment]
@AndyScherzinger Added missing license information for `preview-service-worker.js` (MIT as generated by the workbox plugin (MIT) from workbox sources (MIT) both from Google LLC).

**Pull request merged**